.github/workflows/ci.yml

```diff
@@ -72,6 +73,10 @@ jobs:
           ]
         include:
           - type: gradle-mac
+            command: bin/test.sh 'slow/testOnly -- tests.mill.*'
+            name: Mill Windows integration
+            os: windows-latest
+          - type: mill-windows
```

Comment:
seems like should be other way around
gradle-mac => -- tests.gradle.*'
mill-windows => -- tests.mill.*'

Reply:
unless you don't plan to merge that, then nvm :P

Reply:
Haha, yeah,  it's just for testing though.